composites/OnboardingWizard/helpers/ajaxHelper.js

```diff
@@ -87,6 +87,25 @@ var parseHeaders = ( type, config ) => {
 };
 
 /**
+ * @summary Takes the target object and overwrites fields that are undefined
+ * 			or empty with the defaults object values
```

Comment:
This is one of the very few cases where indentation should be done with spaces. Please change tabs to spaces.